angular-trello/src/app/components/trello-board/trello-board.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component, OnInit } from '@angular/core';
+import { TrelloService } from '../../services/trello.service';
+import { TaskList } from 'src/app/models/taskList.model';
+@Component({
+  selector: 'app-trello-board',
+  templateUrl: './trello-board.component.html',
+  styleUrls: ['./trello-board.component.scss']
+})
+export class TrelloBoardComponent implements OnInit {
+  lists: {[path:string]: TaskList};
+
+  constructor(private storageService: TrelloService) { }
```

Comment:
Budi konzistentan sa imenovanjem promenljivih i imenom klase. Da li se service bavi storage-om, ili Trello Karticama? To su dva različita nivoa apstrakcije. Nije jasno iz ove linije na koji nivo apstrakcije je tvoja komponenta zavisna. (Ja bih predlozio da bude TrelloService ili jos jasnije `TrelloCardService`, a da se interno taj service oslanja na storage service.